src/styles/card.css

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__img {
+  height: 134px;
+  width: 160px;
+  box-sizing: border-box;
+}
+
+.card__description {
+  font-weight: 500;
+  font-size: 12px;
+  color: var(--color-navy);
+  line-height: 18px;
+}
+
+.card__description,
+.card__code,
+.card__stars,
+.card__price,
+.card__button {
+  margin: 0 16px;
+}
+
+.card__item:first-child {
+  margin: 0 19px;
+  padding: 32px 0 36px;
```

Comment:
check comment: you should apply paddings to rhe .card selector, not here
https://github.com/mate-academy/layout_product-cards/pull/2365#discussion_r1056790425

Reply:
And you have to check this question.